src/plugins/intel_npu/tools/single-image-test/main.cpp

```diff
@@ -1302,7 +1304,7 @@ bool testPSNR(const TensorMap& outputs, const TensorMap& references, const int d
 
     auto result = utils::runPSNRMetric(actOutput, refOutput, dstHeight, dstWidth, scaleBorder, normalizedImage);
 
-    if (std::fabs(result - FLAGS_psnr_reference) > FLAGS_psnr_tolerance) {
+    if (FLAGS_psnr_reference - result > FLAGS_psnr_tolerance) {
```

Comment:
Do we expect a negative value here?

Reply:
Yes, it can happen that PSNR metric to provide value greater then the FLAGS_psnr_reference which means that the NPU  actual result beats the reference. For this metric, the larger the result is the better accuracy is.

Reply:
Thanks for the explanation!